python/GafferUI/SpreadsheetUI/_PlugTableView.py

```diff
@@ -417,6 +418,16 @@ def __buttonDoubleClick( self, widget, event ) :
 
 		return True
 
+	def __keyPress( self, widget, event ) :
+
+		if event.modifiers == event.Modifiers.None_ :
+
+			if event.key == "Return" :
+				self.__editSelectedPlugs()
```

Comment:
When editing via keyboard only, it's definitely confusing that the widget pops up where the mouse is and not over the selected cell that your eyes are already focussed on.

Reply:
Yeah, maybe we need to do it differently for keyboard navigation (the other comment on this).

Reply:
Hopefully [this](https://github.com/GafferHQ/gaffer/pull/4001/commits/f0976fe6f0f12e2ab64dfdc12087971c4a6d6af9) is better.  See comments in other section.